lib/common/sync.sh

```diff
@@ -56,13 +60,15 @@ export -f get_lftp_deletions
 _parse_lftp_file_addition_line() {
     local line
     while read line; do
+        local path_dir=`echo $line | grep -o '/.*' | cut -d ' ' -f1`
         line=(${line// / })
         # sample line is:
         # get -O /tmp/argo/dac/csio/2900322 ftp://ftp.ifremer.fr/ifremer/argo/dac/csio/2900322/2900322_Rtraj.nc
+        # get -e -O /tmp/argo/dac/csio/2900322 ftp://ftp.ifremer.fr/ifremer/argo/dac/csio/2900322/2900322_Rtraj.nc
         # we take the basename from the `ftp://` part and add it to the
         # /tmp/argo/dac/... part
-        local path_basename=`basename ${line[3]}`
-        echo ${line[2]}/$path_basename
+        local path_basename=`basename ${line[-1]}`
+        echo $path_dir/$path_basename
```

Comment:
Why not use ${line[-2]} as per the previous line - using 2 methods of cutting up the source line here which means more work deciphering what you are doing  - I'd use one or the other not both